pages/snoopy.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Snoopy</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/snoopy.css">
+</head>
+
+<body class="flex-container">
+
+  <nav class="col-1">
+      <ul>
+        <li>
+          <h2><a href="index.html">Home</a></h2>
```

Comment:
Try to use `h2` and `h3` when you're organizing your content hierarchically. For these, you'd probably have a more accessible page by removing the `h2` and `h3`s and doing the styling with classes on the `li` elements.